README.md

```diff
@@ -429,82 +430,64 @@ specific use cases.
 
 # <a name="implementing-ecs"></a>Implementing ECS
 
-## Adhere to ECS
-
-The following rules apply if an event wants to adhere to ECS
+## Guidelines
 
 * The document MUST have the `@timestamp` field.
 * The [data type](https://www.elastic.co/guide/en/elasticsearch/reference/6.2/mapping-types.html) defined for an ECS field MUST be used.
 * It SHOULD have the field `event.version` to define which version of ECS it uses.
+* As many fields as possible should be mapped to ECS.
 
-To make the most out of ECS as many fields as possible should be mapped to ECS.
-
-## Rules
-
-ECS follows the following writing and naming rules for the fields. The goal of
-these rules is to make the fields easy to remember and have a guide when new
-fields are added.
-
-Often events will contain additional fields besides ECS. These can follow the
-the same naming and writing rules but don't have to.
-
-**Writing**
+**Writing fields**
 
 * All fields must be lower case
```

Comment:
Contributing Guide work is up next.  What do you think about sharing this content (from one source) between the ECS readme and contributing guidelines?  It's obvious why this info belongs in Contributing.  I like it here, too, because knowing the rules makes it easier to remember or guess field names.

Reply:
Problem with this at the moment is that Markdown does not support this AFAIK. So we would need a script (which should be doable). Feel free to copy it over for now and I will automate it then.

Should we potentially move to asciidoc?

Reply:
Sounds like a plan.  I figured it was possible with scripts when I saw how you compiled the README.md file.  

On not sure what moving to asciidoc would buy us.  Until we have a compelling need, I see no reason to move.

Reply:
Asciidoc would allow us to "include" files and leave comments in the docs. But for now I also prefer Markdown especially as most people are more use to it.